api/end_device.proto

```diff
@@ -80,12 +81,18 @@ message MACParameters {
   DataRateIndex rx2_data_rate_index = 11;
   // Frequency for Rx2 (Hz).
   uint64 rx2_frequency = 12;
+  // Maximum uplink duty cycle (of all channels).
+  AggregatedDutyCycle max_duty_cycle = 13;
```

Comment:
No functional changes in this message, moved things around for better allignment with `MACSettings` and clarity